api/v1beta1/aerospikecluster_validating_webhook.go

```diff
@@ -1088,30 +1100,42 @@ func validateNsConfUpdate(
 				}
 
 				// storage-engine update not allowed for now
-				storage, ok1 := singleConf["storage-engine"]
-				oldStorage, ok2 := oldSingleConf["storage-engine"]
+				storage, ok1 := singleConf["storage-engine"].(map[string]interface{})
+				oldStorage, ok2 := oldSingleConf["storage-engine"].(map[string]interface{})
```

Comment:
I would avoid modifying these lines to reduce source of future conflicts with upstream

Reply:
Done